artifacts/snip-plugins-production-readiness.adoc

```diff
@@ -0,0 +1,13 @@
+[id='snip-plugins-production-readiness_{context}']
+[NOTE]
+====
+The following plugins have currently been vetted for RBAC/Audit logging capabilities: 
+
+* RBAC
```

Comment:
so the RBAC plugin has RBAC support? this seems circular...

Reply:
Updated text resolves this circular reference.

Reply:
This topic is not needed anymore, please remove